client/hotrod-client/src/main/java/org/infinispan/client/hotrod/configuration/AbstractConfigurationChildBuilder.java

```diff
@@ -56,6 +57,11 @@ public ConfigurationBuilder balancingStrategy(FailoverRequestBalancingStrategy b
       return builder.balancingStrategy(balancingStrategy);
```

Comment:
Same problem here, this old, deprecated method should take FQN of old `org.infinispan.client.hotrod.impl.transport.tcp.FailoverRequestBalancingStrategy`